dist/images/Dockerfile.test

```diff
@@ -0,0 +1,14 @@
+FROM alpine:edge
+
+RUN set -ex \
+    && echo https://dl-cdn.alpinelinux.org/alpine/edge/testing >> /etc/apk/repositories \
+    && apk update \
+    && apk upgrade \
+    && apk add --no-cache \
+    bash iproute2 iptables iputils tcpdump conntrack-tools \
+    redis nginx qperf iperf3 net-tools curl
```

Comment:
Is redis/tcpdump/conntrack-tools necessary?

Reply:
Will use it later to collect more metrics